docs/compute_resources.md

```diff
@@ -0,0 +1,173 @@
+configured.
+
+Although limits can only be specified on individual containers, it is convenient to talk about pod
+resource limits.
+A *pod resource limit* for a particular resource type is the sum of the resource limits of that type for each
+container in the pod, with unset values treated as zero.
+
+The following pod has two containers.  Each has a limit of 0.5 core of cpu and 100MB of memory.
+The pod can be said to have a limit of 1 core and 200MB of memory.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: frontend
+spec:
+  containers:
+  - name: db
+    image: mysql
+    resources:
+      limits:
+        memory: "100M"
+        cpu: "500m"
+  - name: wp
+    image: wordpress
+    resources:
+      limits:
+        memory: "100M"
+        cpu: "500m"
+```
+
+## How Pods with Resource Limits are Scheduled
+
+When a pod is created, the kubernetes scheduler selects a node for the pod to run on.  A node has a
+capacity: the amount of CPU and memory it can provide for pods.  The scheduler ensures that, for
+each resource type (CPU and memory), the sum of the resource limits of the containers scheduled to
+the node is less than the capacity of the node.  Note that although actual memory or CPU resource
+usage on nodes is very low, the scheduler will still refuse to place pods onto nodes if the capacity
+check fails.  This protects against a resource shortage on a node when resource usage later increases,
+such as due to a daily peak in request rate.
+
+If a pod does not specify a resource limit, then the scheduler treats the limit as zero.
+The scheduler also tries to spread pods evenly across nodes, among other scheduling
+considerations.  As a result, even if pods do not specify resource limits, the system may
+still function fine.  
+
+## How Pods with Resource Limits are Run
+
+When kubelet starts a container of a pod, it passes the CPU and memory limits to the container
+runner (Docker or rkt).
+
+When using Docker:
+- The `spec.container[].resources.limits.cpu` is multiplied by 1024, converted to an integer, and
+  used as the value of the [`--cpu-shares`](
+  https://docs.docker.com/reference/run/#runtime-constraints-on-resources) flag to the `docker run`
+  command.
+- The `spec.container[].resources.limits.memory` is converted to an integer, and used as the value
+  of the [`--memory`](https://docs.docker.com/reference/run/#runtime-constraints-on-resources) flag
+  to the `docker run` command.
+
+**TODO: document behavior for rkt**
+
+If a container exceeds its memory limit, it may be terminated.  If it is restartable, it will be
+restarted by kubelet, as will any other type of runtime failure.  If it is killed for exceeding its
+memory limit, you will see the reason `OOM Killed`, as in this example:
+```
+$ kubectl get pods/memhog
+NAME      READY     REASON       RESTARTS   AGE
+memhog    0/1       OOM Killed   0          1h
+```
+
+A container may or may not be allowed to exceed its CPU limit for extended periods of time.
```

Comment:
The term throttled implies to me use of [CFS Bandwidth Control] (https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt), which we don't use yet.   So I would like to avoid that term.  

And, I don't think that all kubernetes implementation have to let the container exceed its limit if there is nothing else on the node, so I'd rather not say that either.